packages/app/src/utils/dynamicUI/getDynamicRootConfig.ts

```diff
@@ -1,7 +1,6 @@
+import { DynamicRootConfig } from '@internal/plugin-utils';
 import { getScalprum } from '@scalprum/core';
 
-import { DynamicRootConfig } from '../../components/DynamicRoot/DynamicRootContext';
-
 function getDynamicRootConfig(): DynamicRootConfig {
```

Comment:
Probably we can remove it because it is used in .tsx files. Instead we can use the new hook